src/main/resources/application.properties

```diff
@@ -57,7 +57,7 @@ auditing.deployment.name=hocs-info-service
 aws.sqs.access.key=12345
 audit.aws.sqs.access.key=12345
 
-user.email.whitelist=homeoffice.gov.uk,hmpo.gov.uk,cluster2security.gov.uk
+user.email.whitelist=${user.addition.allowed.domains:}
```

Comment:
This naming makes it seem like it's adding to the existing list of allowed domains, but it's actually overwriting it. Something like `$ALLOWED_EMAIL_DOMAINS_FOR_USER_CREATION` or something idk?

Reply:
Change to that - much better - will do the corresponding change to the kube repo.